test/TestAsset/ReflectionTestClass.php

```diff
@@ -55,4 +56,5 @@ protected function _one(string $arg1, ?array $arg2 = null): string
     public static function two(string $arg1, string $arg2)
     {
     }
+    // phpcs:enable
```

Comment:
Same comment here: update the asset to fix CS, versus ignoring CS.

Reply:
Change provided.